### ISSUE TITLE
Crash when using the 1.40.1 : unsupported protocol version: 1.19.3 

### ISSUE THREAD

**Issue by vincss** [PrismarineJS/node-minecraft-protocol#1073]
[X] The [FAQ](https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/docs/FAQ.md) doesn't contain a resolution to my issue 

## Versions
 - minecraft-protocol: 1.40.1
 - server: vanilla/spigot/paper 1.19.3
 - node: 18.12.1

## Detailed description of a problem
I updated the library to the latest 1.40.1 and when I try to start my server, it throws an exception :

> unsupported protocol version: 1.19.3 

## Current code
```js
https://github.com/vincss/mcsleepingserverstarter/blob/master/src/sleepingMcJava.ts#L25
```

## Expected behavior
The server starts and is compatible with version 1.19.3

## Additional context
/

**Comment by jojomatik** [comment]
I've just experienced the same. I believe this happens as the transitive depency `minecraft-data` has not been updated automatically (it has not been updated in `package.json`).

Try `npm update minecraft-data` to fix it.

I will test this out and get back to you soon.

**Comment by jojomatik** [comment]
Seems to work properly after bumping that transitive dependency. @rom1504, could you please consider bumping dependencies, at least if they are absolutely required to use the library. (Related discussion: https://github.com/PrismarineJS/node-minecraft-protocol/pull/1062#issuecomment-1385078542).

Should I create a PR to do so?

**Comment by vincss** [comment]
Hi !
Thanks for the quick fix, it's good for me with the 1.40.2.
